lib/static/modules/reducers/grouped-tests/helpers.js

```diff
@@ -14,7 +14,7 @@ export function handleActiveResults({tree = {}, viewMode = ViewMode.ALL, filtere
         const browser = tree.browsers.byId[result.parentId];
         const testName = browser.parentId;
 
-        if (!isTestNameMatchFilters(testName, testNameFilter, strictMatchFilter)) {
+        if (!isTestNameMatchFilters(testName, browser.name, testNameFilter, strictMatchFilter)) {
```

Comment:
Why?

Reply:
To be able to search for the test including browser name. Because in new UI when you click on copy, the whole title is copied, including browser name.